src/lseq.rs

```diff
@@ -0,0 +1,255 @@
+//! # LSEQ
```

Comment:
isn't it more idiomatic to have `lseq/mod` rather than `lseq.rs + lseq/` ?

Reply:
They changed it in Rust 2018 https://doc.rust-lang.org/nightly/edition-guide/rust-2018/module-system/path-clarity.html#no-more-modrs

Reply:
terrific i never liked having `mod.rs`